Procfile

```diff
@@ -1,2 +0,0 @@
-web: env bin/rails s
-worker: bundle exec sidekiq -q sms -q mailers
```

Comment:
We actually need this Procfile to be able to run the app using `foreman start` on Mac. I would suggest adding it to `.cfignore` if it isn't compatible with CF.